frontend/packages/kubevirt-plugin/src/components/vms/vmi-details-page.tsx

```diff
@@ -0,0 +1,117 @@
+  VirtualMachineInstanceMigrationModel,
+  VirtualMachineInstanceModel,
+  VirtualMachineModel,
+} from '../../models';
+import { getResource } from '../../utils';
+import {
+  VM_DETAIL_OVERVIEW_HREF,
+  VM_DETAIL_DISKS_HREF,
+  VM_DETAIL_NETWORKS_HREF,
+  VM_DETAIL_CONSOLES_HREF,
+} from '../../constants';
+import { VMEvents } from './vm-events';
+import { VMConsoleFirehose } from './vm-console';
+import { VMDetailsFirehose } from './vm-details';
+import { vmiMenuActionsCreator } from './menu-actions';
+import { VMDashboard } from './vm-dashboard';
+
+import './vmi-details-page.scss';
+
+export const VirtualMachinesInstanseDetailsPage: React.FC<VirtualMachinesDetailsPageProps> = (
+  props,
+) => {
+  const { name, ns: namespace } = props.match.params;
+
+  const breadcrumbsForVMPage = (match: any) => () => [
+    {
+      name: VirtualMachineModel.labelPlural,
+      path: `/k8s/ns/${match.params.ns || 'default'}/virtualmachines`,
+    },
+    { name: `${match.params.name} Details`, path: `${match.url}` },
+  ];
+
+  const dashboardPage = {
+    href: '', // default landing page
+    name: 'Dashboard',
+    component: VMDashboard,
+  };
+
+  const overviewPage = {
+    href: VM_DETAIL_OVERVIEW_HREF,
+    name: 'Overview',
+    component: VMDetailsFirehose,
+  };
+
+  const consolePage = {
+    href: VM_DETAIL_CONSOLES_HREF,
+    name: 'Consoles',
+    component: VMConsoleFirehose,
+  };
+
+  const nicsPage = {
+    href: VM_DETAIL_NETWORKS_HREF,
+    name: 'Network Interfaces',
+    component: VMNics,
+  };
+
+  const disksPage = {
+    href: VM_DETAIL_DISKS_HREF,
+    name: 'Disks',
+    component: VMDisksFirehose,
+  };
+
+  const pages = [
+    dashboardPage,
+    overviewPage,
+    navFactory.editYaml(),
+    consolePage,
+    navFactory.events(VMEvents),
+    nicsPage,
+    disksPage,
+  ];
+
+  const resources = [
+    getResource(PodModel, { namespace, prop: 'pods' }),
+    getResource(VirtualMachineInstanceMigrationModel, { namespace, prop: 'migrations' }),
+  ];
+
+  return (
+    <DetailsPage
+      {...props}
+      name={name}
+      namespace={namespace}
+      kind={VirtualMachineInstanceModel.kind}
+      kindObj={VirtualMachineInstanceModel}
+      menuActions={vmiMenuActionsCreator}
+      pages={pages}
+      resources={resources}
+      breadcrumbsFor={breadcrumbsForVMPage(props.match)}
+    >
+      <HintBlock
+        className="kubevirt-details-page__hint-block"
+        title={`Virtual Machine Instanse ${name}`}
+      >
+        <p>
+          This is a VirtualMachineInstance overview page. Please consider using a VirtualMachine
+          that will provide additional management capabilities to a VirtualMachineInstance inside
+          the cluster.
+        </p>
+        <ExternalLink
+          href="https://kubevirt.io/user-guide/docs/latest/architecture/virtual-machine.html"
+          text="Learn more"
+        />
+      </HintBlock>
+    </DetailsPage>
+  );
+};
+
+export type VirtualMachinesDetailsPageProps = {
```

Comment:
please change the name as it is a duplicate

Reply:
done